apps/src/templates/rubrics/EvidenceLevelsForTeachersV2.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {aiEvaluationShape, evidenceLevelShape} from './rubricShapes';
+import {
+  BodyThreeText,
+  BodyFourText,
+  StrongText,
+  Heading6,
+} from '@cdo/apps/componentLibrary/typography';
+import {
+  UNDERSTANDING_LEVEL_STRINGS,
+  UNDERSTANDING_LEVEL_STRINGS_V2,
+} from './rubricHelpers';
+
+export default function EvidenceLevelsForTeachersV2({
+  evidenceLevels,
+  understanding,
+  radioButtonCallback,
+  canProvideFeedback,
+  isAutosaving,
+  aiEvalInfo,
+}) {
+  const passFail = useMemo(() => {
+    if (!!aiEvalInfo) {
+      if (understanding >= 0) {
```

Comment:
Can you help me understand what this is for?

Reply:
Is this for if an "understanding" level has already been assigned by the teacher - then it will return an empty list so that none of the buttons are highlighted, is that correct?

Reply:
Yes, that's right! I'll add some comments.